.github/workflows/terraform-ci.yml

```diff
@@ -76,6 +80,7 @@ jobs:
         aws configure --profile lambda_user set aws_access_key_id "${{ secrets.AWS_ACCESS_KEY_ID }}"
         aws configure --profile lambda_user set aws_secret_access_key "${{ secrets.AWS_SECRET_ACCESS_KEY }}"
         aws configure set role_arn "${{ secrets.AWS_ROLE_ARN }}"
+        aws configure set region us-west-2
```

Comment:
Should use a GH variable to set the region. Then we can pass it to `terraform apply` and `terraform destroy`